File: backend/app/models/review.py
"""backend/app/models/review.py
Pydantic models for Investigator Reviews.
"""
from typing import Literal
from pydantic import BaseModel

class ReviewBase(BaseModel):
    analysis_id: str
    edge_source: str
    edge_target: str
    status: Literal["추가 조사 필요", "특이사항 없음", "보류", ""]
    memo: str

class ReviewCreate(ReviewBase):
    pass

class ReviewResponse(ReviewBase):
    id: int
    created_at: str

File: frontend/src/types/review.ts
// frontend/src/types/review.ts
export interface Review {
  analysis_id: string;
  edge_source: string;
  edge_target: string;
  status: "추가 조사 필요" | "특이사항 없음" | "보류" | "";
  memo: string;
}
